app/models/suggestion.rb

```diff
@@ -0,0 +1,14 @@
+class Suggestion < ApplicationRecord
+  validates :title, :description, presence: true
+
+  belongs_to :user
+  has_many :votes, as: :votable
+
+  def vote_sum
+    total = 0
+    votes.each do |vote|
+      total += vote.vote
```

Comment:
Even better, let ActiveRecord calculate the vote total:
```ruby
suggestion = Suggestion.first
total = suggestion.votes.sum(:vote)
```
This will be much faster since it is a SQL query utilizing COUNT() under the hood.